libdevice/imf_wrapper.cpp

```diff
@@ -0,0 +1,162 @@
+//==----- imf_wrapper.cpp - wrappers for intel math library functions ------==//
```

Comment:
Maybe it would be more clear to name this file as imf_wrapper_fp32.cpp. And move all float16 related stuff to another file imf_wrapper_fp16.cpp.